ui-modules/blueprint-composer/app/components/util/d3-blueprint.js

```diff
@@ -774,18 +774,38 @@ export function D3Blueprint(container, options) {
     function drawRelationships() {
         showRelationships();
 
+        // Generates unique relation ID string.
+        const getRelationId = (relationDataItem) => (relationDataItem.source._id + '_related_to_' + relationDataItem.target._id);
+
+        // Group relationships per direction, as a key.
+        let relationshipsPerDirection = new Map();
```

Comment:
Do we need strict order iteration for `relationshipsPerDirection`? If not, it might be simpler to use a POJO `{}` instead.

Reply:
Agree, can be a POJO instead of Map. Applied in 49f0dff9a3eab634d497228a6a4b339e7583eb22.